.github/workflows/pytest-cov.yaml

```diff
@@ -0,0 +1,40 @@
+name: Pytest Coverage
+on: 
+  push:
+    branches: [ develop ]
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  run_pytest_coverage:
+    runs-on: ubuntu-latest
+    name: Run Pytest Coverage
+    strategy:
+      matrix:
+        python: ["3.7", "3.8", "3.9", "3.10"]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python }}
+
+      - name: Install dependencies
+        run: |
+          pip install --upgrade pip
+
+      - name: Install pytest-cov
```

Comment:
Any dependencies we need should exist in setup.cfg.

https://github.com/ufs-community/workflow-tools/blob/32aafc7d5d29714b09a8966f19583dd25d213403/setup.cfg#L52

`pytest-cov` already exists.  You can install `dev` dependencies using:

```pip install -e .[dev]```